x-pack/plugins/security_solution/public/explore/components/risk_score/translations.ts

```diff
@@ -41,22 +49,24 @@ export const ENTITY_RISK_LEVEL = (riskEntity: RiskScoreEntity) =>
   });
 
 export const getRiskEntityTranslation = (
-  riskEntity: RiskScoreEntity,
+  riskEntity?: RiskScoreEntity,
   lowercase = false,
   plural = false
 ) => {
-  if (lowercase) {
-    if (plural) {
-      return (riskEntity === RiskScoreEntity.host ? HOSTS : USERS).toLowerCase();
-    }
+  const text = ((e) => {
```

Comment:
Nit: To prevent losing the nice variable name `riskEntity` in this code, you can immediately invoke without the parameter. (Don't apply this suggestion directly in GitHub, as GitHub isn't smart enough to allow for multi-line suggestions if any lines of code were deleted 😭 )

```suggestion
   const text = (() => {
    if (!riskEntity && plural) return ENTITIES;
    if (!riskEntity) return ENTITY;

    if (riskEntity === RiskScoreEntity.host && plural) return HOSTS;
    if (riskEntity === RiskScoreEntity.host) return HOST;

    if (riskEntity === RiskScoreEntity.user && plural) return USERS;
    if (riskEntity === RiskScoreEntity.user) return USER;

    return '';
  })();
```

Reply:
Done